### DISCUSSION TITLE
How can I assign new tasks to a project when a vimwiki file containing tasks is opened?

### DISCUSSION THREAD

**Discussion by sdondley** [tbabej/taskwiki#438] [Community support]
To save time on inputting a long series of tasks I do over and over, I generate the task list from a template and save it to a markdown file. What I'd then like to do is open the file and have taskwiki add the tasks to the taskwarrior database. 

However, it doesn't appear to be possible to assign these tasks to a project.

I tried:

```
# Heading | project:some_project
* [ ] task 1
* [ ] task 2
````

When the file is opened for the first time, the tasks just disappear into oblivion. 

Next I tried:

```
* [ ] some task -- project:some_project
```

While this will create `some task` in the taskwarrior database when the file is opened, the task does not get assigned to a project. 

Is there any kind of workaround for what I want to accomplish?

**Comment by liskin** [comment]
This is probably because taskwiki only loads when opening a file, not saves. What might work is using vims `:r` command to read a file into the current buffer, and then saving it.

(But then if you're automating this, you might just manipulate the taskwarrior database directly using `task`)

**Comment by sdondley** [comment]
Ah, excellent idea!

I'm scripting dozens of tasks with lots of complex dependencies. Creating, maintaining and updating that script would be a nightmare. Hoping taskwiki can save me a lot of time.

**Comment by sdondley** [comment]
Thanks @liskin, your suggestion worked perfectly.
